app/src/main/java/com/adriandevs/brunch/presentation/ui/events/adapter/EventViewHolder.kt

```diff
@@ -0,0 +1,55 @@
+package com.adriandevs.brunch.presentation.ui.events.adapter
+
+import androidx.recyclerview.widget.RecyclerView
+import com.adriandevs.brunch.databinding.ItemEventBinding
+import com.adriandevs.brunch.domain.model.Event
+
+class EventViewHolder(binding: ItemEventBinding) :
+    RecyclerView.ViewHolder(binding.root) {
+    private val viewBinding = binding
+
+    fun render(
+        event: Event,
+        listener: EventListener
+    ) {
+        viewBinding.apply {
```

Comment:
You are using viewbinding apply but in configbutton and configticketbutton you are accessing to binding directly. It's not better to pass the button by param or just do a private extension?